system_metrics_collector/src/system_metrics_collector/linux_process_cpu_measurement_node.cpp

```diff
@@ -49,6 +51,24 @@ LinuxProcessCpuMeasurementNode::LinuxProcessCpuMeasurementNode(
 {
 }
 
+LinuxProcessCpuMeasurementNode::LinuxProcessCpuMeasurementNode(
+  const rclcpp::NodeOptions & options)
+: PeriodicMeasurementNode(
+    "linuxProcessCpuCollector",
+    system_metrics_collector::kDefaultCollectPeriod,
+    system_metrics_collector::kStatisticsTopicName,
+    system_metrics_collector::kDefaultPublishPeriod),
+  metric_name_(std::to_string(GetPid()) + kMetricName)
+{
+  const auto r = rcutils_logging_set_logger_level(get_name(), RCUTILS_LOG_SEVERITY_DEBUG);
```

Comment:
Setting the log to debug isn't needed. Let's allow a user to set that configuration and remove our testing code.

Reply:
Done.